test/test_page.rb

```diff
@@ -56,7 +60,7 @@
   end
 
   test "nested url_path" do
-    page = @wiki.page('Eye-Of-Sauron')
+    page = @wiki.page('/Mordor/Eye-Of-Sauron')
```

Comment:
Here you use an absolute instead of relative path. Does it matter? Should we be consistent in our use?

Reply:
I was thinking it would be intuitive to not require the leading slash for a path to start with root, but let it also work with an explicit slash. Is that ok?

Reply:
Yes, but it would be good to add a note in the tests to explain this and a remark in the release notes as well.

Reply:
👍